app/utilities/mailing.py

```diff
@@ -6,22 +6,48 @@ async def send_verification_mail(db_account):
     data = {
         'from': 'FastSurvey <noreply@admin.fastsurvey.io>',
         'to': db_account["email"],
-        'subject': 'Please verify your FastSurvey account',
-        'html': generate_verification_email_html(db_account),
+        'subject': 'Verify your FastSurvey account',
+        'html': verification_email_html(db_account),
         'o:testmode': ENVIRONMENT == 'testing'
     }
     response = await httpx_client.post('/messages', data=data)
     assert(response.status_code == 200), f"{response}"
 
 
-def generate_verification_email_html(db_account):
+def verification_email_html(db_account):
     assert("email_token" in db_account)
     verification_url = (
-        ADMIN_FRONTEND_URL + "/verification?token=" + db_account["email_token"]
+        ADMIN_FRONTEND_URL + "/verify-email?token=" + db_account["email_token"]
     )
     return (
         '<h1>Welcome to FastSurvey!</h1>' +
-        '<p>Please verify this email vie the following link:</p>' +
+        '<p>Please verify this email via the following link:</p>' +
+        f'<a href="{verification_url}" target="_blank">{verification_url}</a>' +
+        '<p>Best,<br/>The FastSurvey Team</p>' +
```

Comment:
I think the correct spelling is: _Best, the FastSurvey team_

Reply:
Yes, that is better!